Kick the Sheets/Views/CalendarView/CalendarCell.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CalendarCell.swift
+//  Kick the Sheets
+//
+//  Created by Ayren King on 9/1/23.
+//
+
+import SwiftUI
+
+struct CalendarCell: View {
+    let date: Date
+    let monthPosition: MonthPosition // this is in reference to what month is being displayed
+    @Binding var selectedDate: Date
+
+    @EnvironmentObject var appState: AppState
+
+    var foregroundColor: Color {
+        guard monthPosition == .current else {
+            return KTSColors.text.color.opacity(0.3)
+        }
+
+        return KTSColors.text.color
+    }
+
+    var backgroundColor: Color {
+        date.backgroundColor(appState.days)
+    }
+
+    var borderColor: Color {
```

Comment:
Even though this file is small, follow conventions and place computed stuff in an extension